src/main/java/org/sonar/FileOutputDemo.java

```diff
@@ -0,0 +1,37 @@
+package org.sonar;
+
+import java.io.*;
+
+class FileOutputDemo 
+{   
+
+ //   private FileOutputDemo(){    
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)